Added ARM64

FalcoG opened the pull request:
This will add support for arm64 and also display the architecture during the npm install phase.

Should resolve https://github.com/customrealms/cli/issues/4

FalcoG committed:
Added ARM64

connerdouglass commented:
I think one more thing is needed. `.goreleaser.yml` needs to have a build section added for `darwin-arm64`. You can just copy and paste the`darwin-amd64` one and replace `amd64` with `arm64`, then you're (probably) good to go!

FalcoG committed:
Add darwin arm64 to the GoReleaser

FalcoG commented:
Got it, config should now be good - ran it locally and it made the necessary files in dist/.

Also tested the binary by using ``go build -o crx cmd/crx/main.go && sudo cp crx /usr/local/bin/`` and ``$ crx`` worked just fine.

connerdouglass commented:
@FalcoG Your PR was just published as `@customrealms/cli` version `0.4.10`. Please make sure the following commands work on your Apple Silicon computer:

```sh
npm install -g @customrealms/cli@0.4.10
crx version
```

It should say it's version 0.4.10

Thanks!

FalcoG commented:
@connerdouglass thanks for updating it!

The latest version works;
```
falco@Falcos-Mac-mini ~ % npm install -g @customrealms/cli@0.4.10

added 1 package, changed 1 package, and audited 3 packages in 6s

found 0 vulnerabilities
falco@Falcos-Mac-mini ~ % crx version
customrealms-cli (crx) v0.4.10
```
